internals.Rmd

```diff
@@ -94,9 +582,9 @@ As can be seen, fields and methods are not differentiated in the construction, a
 
 ```{r}
 Me <- ggproto(NULL, Person,
-  first = "Thomas Lin",
-  last = "Pedersen",
-  birthdate = as.Date("1985/10/12")
+              first = "Thomas Lin",
```

Comment:
I think the existing formatting is correct (according to the tidyverse style guide)

Reply:
Fixed. I swear I don't even remember changing it originally!